dart/dynamics/SimpleFrame.cpp

```diff
@@ -120,6 +132,13 @@ SimpleFrame& SimpleFrame::operator=(const SimpleFrame& _otherFrame)
 }
 
 //==============================================================================
+std::shared_ptr<SimpleFrame> SimpleFrame::createChildSimpleFrame(
```

Comment:
I'm very much inclined to agree with your opposition to (2), although I'm not against it quite as strongly :wink: